teamA/lib/Controller/main_controller.dart

```diff
@@ -41,19 +41,20 @@ class MainController {
     isLoggedIn = false;
   }
 
-/* //// original file either daanish code or scott code
-  Future<List<Course>> getCourses() async 
-  {
-    if (courses != []){
+  //// original file either daanish code or scott code
+  Future<List<Course>> getCourses() async {
+    //if (courses != []) {
+    if (courses.isNotEmpty) {
+      // this above line changed by safia because in dart we use isempty instead of !=
       return courses;
     }
     var moodleApi = MoodleApiSingleton();
     try {
       courses = await moodleApi.getCourses();
-      if (courses.isNotEmpty) {
-        courses.removeAt(
-            0); // first course is always "Moodle" - no need to show it
-      }
+      // if (courses.isNotEmpty) { commented by safia in original code because of this first course is getting removed
```

Comment:
Same comments as above

Reply:
Removed the commented code, its good now.